src/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.fsproj

```diff
@@ -1,9 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../Shared/base.props" />
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DebugType>full</DebugType>
+  <PropertyGroup>
+    <!--Determinism of the F# Compiler #1042-->
```

Comment:
I presume this refers to <https://github.com/dotnet/fsharp/issues/1042> but I don't understand why.

Reply:
I Use VS16.2.1 and have the the following build error when i checkout the latest dev branch.

```Severity	Project Rank	Code	Description	Project	File	Line	Suppression State
Error	6	FS2026	Deterministic builds only support portable PDBs (--debug:portable or --debug:embedded)	Microsoft.DocAsCode.Metadata.ManagedReference.FSharp	P:\GitHub\dotnet\docfx\src\Microsoft.DocAsCode.Metadata.ManagedReference.FSharp\FSC	1	Active
```

It refers to https://github.com/dotnet/fsharp/pull/2954/commits/485d35b51860db6d8a49ffaddc3ef1ef42a138e7 


A alternative and also working solution is to set to the old behavior. (I would prefere a Determinictic build)
```
<PropertyGroup>
    <!--Determinism of the F# Compiler #1042-->
    <Deterministic>false</Deterministic>
  </PropertyGroup>
```

I am not a FSharp expert, so tell me what i should do.